su-template/default/layout/html/headers.erb

```diff
@@ -0,0 +1,15 @@
+  <meta charset="<%= charset %>">
+  <!-- VIEWPORT -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Again, from the core template `"default/html/layout/headers.erb"` file. 
**I didn't change it, just added the comment label.** 

> Would this affect the responsiveness for the pages?

I think it has to do with display scaling, and whether the device is a full-sized display, or a small display such as a tablet or phone. **Again, It is HTML5**, so I wonder why it is currently in the core template as he has made it plain he doesn't wish to support HTML5 yet.

See the explanation of the **viewport** meta tag:
[http://www.w3schools.com/tags/tag_meta.asp](http://www.w3schools.com/tags/tag_meta.asp)